be/src/olap/tablet_schema_cache.h

```diff
@@ -52,6 +55,25 @@ class TabletSchemaCache {
         return iter->second;
     }
 
+private:
+    /**
+     * @brief recycle when TabletSchemaSPtr use_count equals 1.
+     */
+    void _recycle() {
+        for (;;) {
+            std::this_thread::sleep_for(
+                    std::chrono::seconds(config::tablet_schema_cache_recycle_interval));
+            std::lock_guard guard(_mtx);
+            for (auto iter = _cache.begin(), last = _cache.end(); iter != last;) {
+                if (iter->second.use_count() == 1) {
+                    iter = _cache.erase(iter);
+                } else {
+                    ++iter;
+                }
+            }
```

Comment:
Recycler does not consider memory consumption and timing. I worry that would lead to performance drop. May be we should have a performance test with a smaller enough recycle_interval.

Reply:
That won't have performance problem，because it just recycle schemas only catched by tablet_schema_cache.